conf/application.conf

```diff
@@ -176,6 +176,7 @@ jsonld {
     type = "sb"
     localPath = "/tmp"
     ttl = 43200
+    sb_schema = ["https://sunbirdstagingpublic.blob.core.windows.net/sunbird-dial-staging/jsonld-schema/local/sb/schema.jsonld"]
```

Comment:
Start adding comments what should be that value & why it is required for the new config properties what we define going forward.